test/test_mpi.jl

```diff
@@ -0,0 +1,21 @@
+using Clang.Generators
+
+@testset "MPI.jl" begin
+    @info "Testing generator for JuliaParallel/MPI.jl"
+    mkpath("build_mpi")
```

Comment:
Following my comments above, I think there is no need to use this.

Reply:
This name is very confusing. You could use another directory name like `gen_mpi`.

Reply:
Solved.